geowave-accumulo/src/main/java/mil/nga/giat/geowave/accumulo/mapreduce/NativeReduceContext.java

```diff
@@ -38,12 +37,20 @@
 		ReduceContext<KEYIN, VALUEIN, GeoWaveInputKey, Object>
 {
 	private final ReduceContext<KEYIN, VALUEIN, GeoWaveInputKey, ObjectWritable> writableContext;
-	private final AdapterStore adapterStore;
+	private final HadoopAdapterStore adapterStore;
```

Comment:
this is probably the most minor nit of them all; this doesn't implement adapter store and delegate to the wrapped adapter, so its name is just a little misleading, I'd say it acts as a helper object doing some heavy lifting with serializers and caching them (ie. I'd call it something like HadoopAdapterSerializationHelper rather than a '*adapterstore').  Take it or leave it, this is minor